DigitalLearningSolutions.Data.Tests/DataServices/UserDataServiceTests/DelegateUserDataServiceTests.cs

```diff
@@ -418,5 +418,57 @@ public void ActivateDelegateUser_activates_delegate_user()
             // Then
             userDataService.GetDelegateUserById(delegateUserId)!.Active.Should().BeTrue();
         }
+
+        [Test]
+        public void SetDelegateUserLearningHubAuthId_correctly_sets_delegates_learningHubAuthId()
+        {
+            using var transaction = new TransactionScope();
+
+            // Given
+            const int delegateId = 3;
+            const int learningHubAuthId = 1234;
+
+            // When
+            userDataService.SetDelegateUserLearningHubAuthId(delegateId, learningHubAuthId);
+
+            // Then
+            var result = userDataService.GetDelegateUserLearningHubAuthId(delegateId);
+
+            result.Should().NotBeNull()
+                .And.Subject.Should().Be(learningHubAuthId);
+        }
+
+        [Test]
+        public void GetDelegateUserLearningHubAuthId_returns_expected_delegate_learningHubAuthId()

```

Comment:
This is the same as the test above but with different values right?

Reply:
Yes it is. Nice. This one was added during markups and although the SIT is different it is essentially duplication.